ztag/schema.py

```diff
@@ -1437,12 +1437,26 @@ class LintBool(String):
                     "tls_1_1": ztag_tls_support,
                     "tls_1_2": ztag_tls_support,
                     "ecdhe": ztag_ecdh,
+                }),
+                "https_www":SubRecord({
+                    "tls":ztag_tls,
+                    "heartbleed":ztag_heartbleed,
+                    "dhe": ztag_dh,
+                    "export_rsa": ztag_rsa_export,
```

Comment:
This should be `rsa_export` and `dhe_export`, see https://github.com/Censys/censys-definitions/blob/master/proto/protocols.proto#L53